4.3/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+if [[ -n "$ADMINER_DESIGN" ]]; then
+    ln -s "designs/$ADMINER_DESIGN/adminer.css" .
```

Comment:
To think about it, not sure on it myself: You `set -e`, but this is a non-critical line that may fail on read only file systems. Consider adding a `|| true`.